doc/lua_api.txt

```diff
@@ -6206,6 +6206,8 @@ object you are working with still exists.
        should appear in first person.
 * `get_attach()`: returns parent, bone, position, rotation, forced_visible, 
     or nil if it isn't attached.
+* `get_children()`: returns children of the object as a table, or nil if there
+    are none.
```

Comment:
```suggestion
* `get_children()`: returns children of the object as a table, or an empty one if there
    are none.
```
Correspondingly to the suggestion below, here it should be fixed.

Reply:
The fact it rhymed bugged me, I put "if there aren't any"